src/eko/io/runcards.py

```diff
@@ -82,6 +111,10 @@ class Configs(DictLike):
     """
     ev_op_iterations: int
     """Number of intervals in which to break the global path."""
+    scvar_method: Optional[ScaleVariationsMethod]
+    """"""
```

Comment:
```suggestion
    """Scale variation method"""
```

Reply:
Done in 3a380883cfe68afb46ba1dcf25baf5611860e129